examples/api/test/widgets/framework/error_widget.0_test.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/framework/error_widget.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+
+  ErrorWidget.builder = (FlutterErrorDetails details) {
+    return ErrorWidget(details.exception);
+  };
```

Comment:
Consider moving these lines inside the first test and the content of `tearDown` inside the second test.

For an example of how to set the builder inside a test and restore the original builder at the end of the test, see https://github.com/flutter/flutter/blob/7a44392f3f17c6c56ef1d292d895c2e98db4b266/packages/flutter/test/widgets/error_widget_builder_test.dart#L9-L26

Reply:
Done @bleroux